.eleventy.js

```diff
@@ -272,6 +288,13 @@ module.exports = function (config) {
   // Make .yml files work in the _data directory.
   config.addDataExtension('yml', (contents) => yaml.safeLoad(contents));
 
+  // Make CSP hashes accessible to firebase config.
+  if (isProd) {
+    config.on('afterBuild', () => {
+      fs.writeFileSync('script-hash-list.json', JSON.stringify([...hashList]));
```

Comment:
Could we generate `firebase.json` in this function to eliminate the need for a temporary file and changes to `npm run dev`?

Reply:
We could, but, ordering in build systems like this has always been a pain.

I don't mind your `npm run dev` change. My main thing would be potentially to put this file inside the `dist/` folder, so it's clear it's an intermediate file and not something real that needs to be committed forever.